samples/nrf9160/lwm2m_client/README.rst

```diff
@@ -482,11 +506,11 @@ Testing
 Firmware Over-the-Air (FOTA)
 ============================
 
-You can update the firmware of the device if you are using Coiote Device Management server as the LwM2M server.
+You can update the firmware of the device if you are using Coiote Device Management server or Leshan Demo Server.
```

Comment:
_or Leshan Server._ would be sufficient here as someone might have set up their own Leshan server instance instead of using the publicly available demo server.

Reply:
fixed